File: OnJeong_BE/src/main/java/com/a503/onjeong/domain/news/News.java
package com.a503.onjeong.domain.news;

import com.a503.onjeong.global.BaseEntity;
import jakarta.persistence.*;

import lombok.*;

@Entity
@Builder
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class News {

    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "news_id")
    private Long Id;
    @Column(name = "news_title")
    private String title;
    @Column(name = "news_category")
    private int category;
    @Column(name = "news_description")
    private String description;
    @Column(name = "news_url")
    private String url;
    @Column(name = "news_image")
    private String image;

    @Builder
    public News(
            Long Id,
            String title,
            int category,
            String description,
            String url,
            String image
    ){
        this.Id = Id;
        this.title = title;
        this.category = category;
        this.description = description;
        this.url = url;
        this.image = image;
    }
}

File: OnJeong_FE/app/src/main/java/com/a503/onjeong/domain/news/News.kt
package com.example.myapplication

import javax.persistence.*

@Entity
data class News(
    @Id
    @javax.persistence.GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "news_id")
    var id: Long? = null,

    @Column(name = "news_title")
    var title: String? = null,

    @Column(name = "news_category")
    var category: Int = 0,

    @Column(name = "news_description")
    var description: String? = null,

    @Column(name = "news_url")
    var url: String? = null,

    @Column(name = "news_image")
    var image: String? = null
)
